package.json

```
@@ -72,6 +66,18 @@
                     "default": "",
                     "description": "Absolute path to Yseop CLI executable (“yseop” or “yseop.bat” − see Yseop CLI doc for more info)."
                 },
+                "yseopml.diagnosticLevel": {
+                    "type": "string",
+                    "scope": "window",
+                    "enum": [
+                        "Error",
+                        "Warning",
+                        "Information",
+                        "Hint"
+                    ],
+                    "default": "Information",
+                    "description": "Issues found by parser are marked with a Diagnostic Severity Level. This affects the color of squiggle."
```

I assume that, just like in Eclipse, the type of highlighting can be configured, so it’s better, I think, not to explicitly refer to “squiggles” or anything.

```suggestion
                    "description": "Issues found by the parser are marked with a Diagnostic Severity Level. This affects the way these issues are highlighted in the code."
```